class/defaults.yml

```diff
@@ -42,3 +42,17 @@ parameters:
           enabled: true
       extraArgs:
         - --dns01-recursive-nameservers="${cert_manager:dns01-recursive-nameservers}"
+      resources:
+        requests:
+          cpu: 50m
+          memory: 512Mi
```

Comment:
I'm not sure if this is a bit excessive, but I've looked at a bunch of cert-manager instances, and most of them appear to consume ~450MiB of memory, so from that perspective this default should make some amount of sense.

Reply:
It's pretty important infrastructure so 🤷‍♀️